src/site/content/en/progressive-web-apps/add-manifest/index.md

```diff
@@ -181,6 +182,45 @@ be made to launch full screen.
   </table>
 </div>
 
+#### `display_override` {: #display-override }
+
+Web apps can choose how they are displayed by setting a `display` mode in their manifest as
+[explained above](#display). Browsers are *not* required to support all display modes, but they
+*are* required to support the
+[spec-defined fallback chain](https://w3c.github.io/manifest/#dfn-fallback-display-mode)
+(`"fullscreen"` → `"standalone"` → `"minimal-ui"` → `"browser"`). If they don't support a given
+mode, they fall back to the next display mode in the chain. This inflexible behavior can be
+problematic in rare cases, for example, a developer cannot request `"minimal-ui"` without being
+forced back into the `"browser"` display mode in case `"minimal-ui"` is not supported by a given
+browser. Another problem is that the current behavior makes it impossible to introduce new display
+modes in a backward compatible way, since explorations like tabbed application mode don't have a
+natural place in the fallback chain.
+
+These problems are solved by the `display_override` property, which the browser considers *before*
+the `display` property. Its value is a sequence of strings that are considered in-order, and the
```

Comment:
```suggestion
the `display` property. Its value is a sequence of strings that are considered in the listed order, and the
```

Reply:
It adds a little more clarity, I think, by disambiguating from the spec defined order of `display`. Do something similar in the other article.